src/Core/FontMeasurementCache.re

```diff
@@ -12,13 +12,14 @@ type t = {
 let create =
     (
       ~fontFamily: Revery.Font.Family.t,
+      ~fontWeight: Revery.Font.Weight.t,
       ~fontSize: float,
       ~features: list(Harfbuzz.feature),
       ~smoothing: Revery.Font.Smoothing.t,
     ) => {
   let loadedFont =
     fontFamily
-    |> Revery.Font.Family.toSkia(Revery.Font.Weight.Normal)
+    |> Revery.Font.Family.toSkia(fontWeight)
```

Comment:
Thanks for working through all this plumbing, @marcagba !

Reply:
I have to say, it felt like I started modifying the code and then just relied on the reason compiler to show me where I had to also modify things! A very pleasant experience when you're used to write js all day ! 😄